src/dashboard/tests/test_filesystem_ajax.py

```diff
@@ -10,6 +10,11 @@
 from django.test.client import Client
 import pytest
 
+try:
+    import mock
```

Comment:
Why is the `try/except` necessary here? I noticed `mock` is included in the `test` and `dev` requirements and other test modules in the `dashboard` package don't have it.

Reply:
Good point, will fix.